src/OpenIddict.Server/OpenIddictServerBuilder.cs

```diff
@@ -175,6 +177,11 @@ public OpenIddictServerBuilder AddEncryptionKey(SecurityKey key)
 
             if (key.IsSupportedAlgorithm(SecurityAlgorithms.Aes256KW))
             {
+                if (key.KeySize != 256)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(key), SR.GetResourceString(SR.ID0283));
```

Comment:
Let's use `InvalidOperationException` to match the other exceptions thrown from this method.